### ISSUE TITLE
Revenir à l'élément

### ISSUE THREAD

**Issue by Drakone** [pixelhumain/co2#472]
"manque la possibilité de cliquer sur un retour direct à la page. Obliger de faire recul de page. exemple: je suis dans espace CO. Je vois dans le bandeau du haut l'image de l'organisation. Ce serait bien de pouvoir y retourner en cliquant dessus. pour y retourner soit je passe en cliquant sur mon compte puis en cliquant sur un post ou je vois mon organisation, soit en cliquant sur recul du navigateur"

https://chat.communecter.org/channel/communecterTesteurs?msg=oATYm3mxWw5e4uGFP

**Comment by hubeez** [comment]
précision suite à mes rques d'hier sur le chat: comme j'explore l'outil sans le connaitre, peut être que ma façon  de l'utiliser n'est pas la bonne. Ceci étant, ce peut être aussi un bon moyen de mesurer l'intuitivité et de voir comment un utilisateur "moyen" s'approprie l'outil en mode "je me débrouille sans lire la doc!"

**Comment by clement59** [comment]
Yes on a tellement changer l'outil dans cette dernière version que la documentation d'avant et devenu complètement obsolète ... du coup, on a tout à refaire ! Et toutes remarques d'ergonomie et d'utilisations sont les bienvenues.
